test_scripts/API/PerformAudioPassThru/020_ATF_P_PerformAudioPassThru_WARNING_Missing_audioPassThruIcon_MandatoryParams_STATIC.lua

```diff
@@ -0,0 +1,124 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PerformAudioPassThru] "audioPassThruIcon" does NOT exist at 'AppStorageFolder'
+-- [HMI API] UI.PerformAudioPassThru request/response
+-- [HMI API] TTS.Speak request/response
+-- [Mobile API] PerformAudioPassThru request/response
+-- [HMI_API] [MOBILE_API] The "audioPassThruIcon" param at "ImageFieldName" struct
+--
+-- Description:
+-- In case mobile app sends PerformAudioPassThru_request to SDL 
+-- with valid <audioPassThruIcon> parameter
+-- and the requested <audioPassThruIcon> does NOT exist at app`s sandbox
+-- and with another related to request valid mandatory params
+-- transfer UI.PerformAudioPassThru (<audioPassThruIcon>, other params)_request + Speak_request (depends on parameters provided by the app) to HMI
+-- respond with <resultCode_received_from_HMI> + <info> to mobile app (expected resultCode from HMI is "WARNINGS" and info: "Reference image(s) not found" 
+-- as described in [CTR] Conditions for SDL to handle case when requested <Image> does not exist at "AppStorageFolder"
+--
+-- 1. Used preconditions
+-- 1.1. PerformAudioPassThru RPC is allowed by policy
+-- 1.2. audioPassThruIcon does not exist in created by app AppStorageFolder
+-- 1.3. Request is sent with valid audioPassThruIcon parameter with type STATIC, all mandatory params used in PerformAudioPassThru_request are present and within bounds 
+--
+-- 2. Performed steps
+-- Send PerformAudioPassThru (audioPassThruIcon, mandatory params)
+-- HMI sends UI.PerformAudioPassThru (WARNINGS, info) to SDL
+-- HMI sends TTS.Speak (SUCCESS)
```

Comment:
@maria-mih, because test using only mandatory parameters for PerformAudioPassThru and they are not related with TTS.Speak, so TTS.Speak should not be sent from SDL to HMI.